scaling_performance/cluster_maximums.adoc

```diff
@@ -102,6 +102,66 @@ application requirements.]
 
 |===
 
+[[scaling-performance-tested-maximums-environment]]
+== Environment and configuration on which {product-title} cluster maximums are tested
+
+Infrastructure as a service provider: OpenStack
+
+[options="header",cols="6*"]
+|===
+|Node |vCPU |RAM(MiB) |Disk size(GiB) |pass-through disk |Count
+
+| Master/Etcd footnoteref:[masteretcdnvme, The master/etcd nodes are backed by NVMe disks as etcd is I/O intensive and latency sensitive.]
+| 16
+| 124672
+| 128
+| Yes, NVMe
+| 3
+
+| Infra footnoteref:[infranodes, Infra nodes host the Router, Registry, Logging and Monitoring and are backed by NVMe disks.]
+| 40
+| 163584
+| 256
+| Yes, NVMe
+| 3
+
+| Cluster DNS
+| 1
+| 1740
+| 71
+| No
+| 1
+
+| Load Balancer
+| 4
+| 16128
+| 96
+| No
+| 1
+
+| Container Native Storage footnoteref:[cns, Container Native Storage or Ceph storage nodes are backed by NVMe disks.]
+| 16
+| 65280
+| 200
+| Yes, NVMe
+| 3
+
+| Jump Node footnoteref:[jumpnode, Jump node is part of the OCP network and is used to orchestrate the performance and scale tests.]
```

Comment:
Bastion Node would be better terminology and more familiar to the audience.

Bastion Node footnoteref:[bastionnode, The Bastion node is part of the OCP network and is used to orchestrate the performance and scale tests.]

Reply:
Fixed it, thanks.